Parfors compile error due to stdint.h missing on Windows Python2.7 VC++9.0

Compilation parfor C files is failing on windows python2.7 due to missing stdint.h

```
c:\users\buildbot\hpa-ci\buildbot-slave\var-64\pr_numba-py2_7-np1_10-w64\build\numba\npyufunc\gufunc_scheduler.h(4) : fatal error C1083: Cannot open include file: 'stdint.h': No such file or directory
```

--- a/numba/npyufunc/gufunc_scheduler.h
+++ b/numba/npyufunc/gufunc_scheduler.h
@@ -1,7 +1,7 @@
 #ifndef GUFUNC_SCHEDULER
 #define GUFUNC_SCHEDULER
 
-/* define int64_t and uint64_t for VC9 */
+/* define int64_t and uint64_t for Visual Studio, where stdint only available > VS2008 */
 #ifdef _MSC_VER
     #define int64_t signed __int64
     #define uint64_t unsigned __int64

--- a/numba/npyufunc/gufunc_scheduler.h
+++ b/numba/npyufunc/gufunc_scheduler.h
@@ -1,11 +1,17 @@
 #ifndef GUFUNC_SCHEDULER
 #define GUFUNC_SCHEDULER
 
-#include <stdint.h>
+/* define int64_t and uint64_t for VC9 */
+#ifdef _MSC_VER
+    #define int64_t signed __int64
+    #define uint64_t unsigned __int64
+#else
+    #include <stdint.h>
+#endif
 
 #ifndef __SIZEOF_POINTER__
     /* MSVC doesn't define __SIZEOF_POINTER__ */
-    #if   defined(_WIN64)
+    #if defined(_WIN64)
         #define intp int64_t
         #define uintp uint64_t
     #elif defined(_WIN32)

--- a/numba/npyufunc/gufunc_scheduler.h
+++ b/numba/npyufunc/gufunc_scheduler.h
@@ -2,7 +2,19 @@
 #define GUFUNC_SCHEDULER
 
 #include <stdint.h>
-#if __SIZEOF_POINTER__ == 8
+
+#ifndef __SIZEOF_POINTER__
+    /* MSVC doesn't define __SIZEOF_POINTER__ */
+    #if   defined(_WIN64)
+        #define intp int64_t
+        #define uintp uint64_t
+    #elif defined(_WIN32)
+        #define intp int
+        #define uintp unsigned
+    #else
+        #error "cannot determine size of intp"
+    #endif
+#elif __SIZEOF_POINTER__ == 8
     #define intp int64_t
     #define uintp uint64_t
 #else

